cuebot/scheduler-sim/priority_starve_watch.py

```
@@ -0,0 +1,147 @@
+
+SQL_AGE = f"""
+SELECT cls, max(EXTRACT(EPOCH FROM (now()-ts_started)))::int FROM (
+  SELECT DISTINCT j.pk_job, {CLS} AS cls, j.ts_started
+  FROM frame f JOIN job j ON f.pk_job = j.pk_job
+  WHERE ({SEL}) AND f.str_state='WAITING'
+) s GROUP BY cls;
+"""
+
+
+def q(sql):
+    return subprocess.run(PSQL + ["-c", sql], capture_output=True, text=True,
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)